nix/tools/bazel-cc-toolchain/default.nix

```diff
@@ -70,5 +70,5 @@ let cc-darwin =
 in
 buildEnv {
   name = "bazel-cc-toolchain";
-  paths = [ customStdenv.cc ] ++ (if stdenv.isDarwin then [ ] else [ binutils ]);
+  paths = [ customStdenv.cc ] ++ (if stdenv.isDarwin then [ darwin.binutils ] else [ binutils ]);
```

Comment:
Is this necessary now? For what?

Reply:
Yes, the old toolchain just picked `/usr/bin/strip` and friends. To make these parts hermetic as well, we want to use nixpkgs provided tools. On MacOS one use to use `darwin.binutils` instead of `binutils`. The latter includes a `bin/strip` that chokes on MACH-O binaries. See https://github.com/NixOS/nixpkgs-channels/blob/10100a97c8964e82b30f180fda41ade8e6f69e41/pkgs/os-specific/darwin/binutils/default.nix#L32